mtgjson4/mtg_builder.py

```diff
@@ -86,8 +88,8 @@ def determine_layout_and_div_name(
         card_info['cmc'] = mtg_parse.parse_card_cmc(soup_oracle, div_name)
 
         # Get other side's name for the user
-        has_other, card_other_name = mtg_parse.parse_card_other_name(soup_oracle, div_name, card_layout)
-        if has_other:
+        card_other_name = mtg_parse.parse_card_other_name(soup_oracle, div_name, card_layout)
+        if card_other_name is not None:
```

Comment:
Note this pattern is much safer; mypy is aware of `is not None` checks and will promote `Optional[A]` to `A` inside the `if` branch